src/components/AddFood.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function AdFood(props){
+    return(
```

Comment:
a simple submit and sending the input values via the event target is ok, however, using in React you could use the state for getting the input values and sending them on the function invocation via parameter. Check the solution sent to see how it is done there.